src/test/java/io/github/thebusybiscuit/slimefun4/api/items/settings/TestMaterialTagSetting.java

```diff
@@ -68,7 +69,7 @@ void testAllowedValue() {
         item.register(plugin);
 
         List<String> tagContents = tag.getValues().stream().map(Material::name).collect(Collectors.toList());
-        Assertions.assertIterableEquals(tagContents, setting.getValue());
+        Assertions.assertTrue(new HashSet<>(tagContents).equals(new HashSet<>(setting.getValue())));
```

Comment:
```suggestion
        Assertions.assertEquals(tagContents, setting.getValue());
```

Reply:
Hi, the point with converting the list to a hashSet here is to avoid comparing the two lists with respect to both elements and their orders. I tested the new line you're suggesting and it appears that `Assertions.assertEquals` still take into account the orders in the list, so the nondex still fails.

Reply:
Yes, my fault for not reading the description of this PR 😄